Documentation/Books/AQL/Functions/Document.md

```diff
@@ -47,6 +47,34 @@ FOR attributeArray IN attributesPerDocument
         RETURN {attr, count}
 ```
 
+CHECK_DOCUMENT()
+----------------
+
+`CHECK_DOCUMENT(document) → checkResult`
+
+Returns true if *document* is a valid document object, i.e. a document without any
+duplicate attribute names. Will return *false* for any non-objects/non-documents,
+and *true* for any objects/documents without duplicate attribute names.
+
+Please note that this is an internal function for validating database objects and
```

Comment:
Should we move it to https://docs.arangodb.com/3.4/AQL/Functions/Miscellaneous.html#internal-functions and reference it here (Document.md)?